[2024-05-07T08:07:13.434+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-07T08:07:13.462+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_parse_weather_data.fetch_and_save_weather_data scheduled__2024-05-03T21:00:00+00:00 [queued]>
[2024-05-07T08:07:13.471+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_parse_weather_data.fetch_and_save_weather_data scheduled__2024-05-03T21:00:00+00:00 [queued]>
[2024-05-07T08:07:13.472+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-07T08:07:13.486+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): fetch_and_save_weather_data> on 2024-05-03 21:00:00+00:00
[2024-05-07T08:07:13.493+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=314) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-07T08:07:13.494+0000] {standard_task_runner.py:63} INFO - Started process 327 to run task
[2024-05-07T08:07:13.494+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fetch_and_parse_weather_data', 'fetch_and_save_weather_data', 'scheduled__2024-05-03T21:00:00+00:00', '--job-id', '548', '--raw', '--subdir', 'DAGS_FOLDER/json_dag.py', '--cfg-path', '/tmp/tmpqp38uqah']
[2024-05-07T08:07:13.496+0000] {standard_task_runner.py:91} INFO - Job 548: Subtask fetch_and_save_weather_data
[2024-05-07T08:07:13.541+0000] {task_command.py:426} INFO - Running <TaskInstance: fetch_and_parse_weather_data.fetch_and_save_weather_data scheduled__2024-05-03T21:00:00+00:00 [running]> on host e8691f41e037
[2024-05-07T08:07:13.632+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_and_parse_weather_data' AIRFLOW_CTX_TASK_ID='fetch_and_save_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-03T21:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-03T21:00:00+00:00'
[2024-05-07T08:07:13.634+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-07T08:07:17.325+0000] {logging_mixin.py:188} INFO - Data has been successfully saved to the JSON file
[2024-05-07T08:07:17.326+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-07T08:07:17.328+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-07T08:07:17.344+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=fetch_and_parse_weather_data, task_id=fetch_and_save_weather_data, execution_date=20240503T210000, start_date=20240507T080713, end_date=20240507T080717
[2024-05-07T08:07:17.402+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-07T08:07:17.428+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-07T08:07:18.028+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
